go-bindata-assetfs/main.go

```diff
@@ -54,16 +51,10 @@ func assetFS() *assetfs.AssetFS {
 	}
 	panic("unreachable")
 }`)
+	// Close files BEFORE remove calls (don't use defer).
```

Comment:
This would leak FDs in principle, but I don't think it matters in practice.

In UNIX there's no problem with closing FDs after deleting the backing file. I take it that windows complains when you do. Is that correct?